rustls/src/server/verifier_builder.rs

```diff
@@ -0,0 +1,427 @@
+use std::{fmt, sync::Arc};
```

Comment:
I made a new module (`src/server/verifier_builder.rs`) for this because `src/verify.rs` is looking a bit busy to me. Arguably this new code could also live there :shrug: